src/area-chart/model/use-chart-model.ts

```diff
@@ -18,6 +18,7 @@ import { throttle } from '../../internal/utils/throttle';
 
 const MAX_HOVER_MARGIN = 6;
 const SVG_HOVER_THROTTLE = 25;
+const POPOVER_DEADZONE = 12;
```

Comment:
Does it make sense to declare it somewhere under src/internal/components/chart-popover?